greenplumpython/table.py

```diff
@@ -141,6 +140,34 @@ def __getitem__(self, *args, **kwargs):  # type: ignore
         """
         return self._getitem(*args, **kwargs)  # type: ignore
 
+    def __repr__(self):
+        """
+        Return a string representation for a table
+        """
+        repr_string = ""
+        ret = list(self.fetch())
+        repr_string += (("| {:10} |" * len(ret[0])).format(*ret[0])) + "\n"
```

Comment:
What will happen when the length of the column names is greater than 10 charaters? It would be great to have some tests to cover the scenario. You don't need to do that in this PR but please try to improve it if you have ideas.

Reply:
Got it, let's add a `FIXME` at the moment